doc/whats_new/v1.4.rst

```diff
@@ -58,3 +58,21 @@ TODO: update at the time of the release.
   :meth:`base.OutlierMixin.fit_predict` now accept ``**kwargs`` which are
   passed to the ``fit`` method of the the estimator. :pr:`26506` by `Adrin
   Jalali`_.
+
+:mod:`sklearn.decomposition`
+............................
+
+- |Fix| :meth:`decomposition.SparseCoder.transform` now preserves data
+  types of transformed input ``X`` based on ``preserve_dtypes`` parameter.
+  :pr:`26691` by :user:`SHREESHA KUMAR BHAT<Shreesha3112>`.
+
+- |Fix| :func:`decomposition.tests.test_dict_learning_online_numerical_consistency`
+  now uses precomputed_dictionary insted of ``X.T`` for ``dictionary`` parameter of
+  :class:`decomposition.SparseCoder`. :pr:`26691` by :user:`SHREESHA KUMAR BHAT<Shreesha3112>`.
+
+:mod:`sklearn.utils`
+....................
+
+- |Enhancement| :func:`utils.estimator_checks._construct_instance` now accepts
```

Comment:
You don't need to have an entry for this change because it is changes internals only.